app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str):
+    command_list = command.split(" ")
+    if command_list[0] != "cp":
+        print("Enter correct command (cp)")
+    elif command_list[1] != command_list[2]:
+        with open(command_list[1], "r") as file:
+            file_lines = file.readlines()
+            with open(command_list[2], "w") as file2:
+                file2.write("".join(file_lines))
```

Comment:
If you join all lines here, it's better to read all file content at the beginning)